docs/development-guide/compatibility.md

```diff
@@ -2,7 +2,7 @@
 
 ## Operating systems
 
-PyQIR runs on most x86-64 operating systems that can run Python 3.6+; however,
+PyQIR runs on most x86-64 operating systems that can run Python 3.7+; however,
```

Comment:
The supported tier 1 and tier 2 need to be updated according to the system we validate as part of the build. jammy, bullseye, etc